v2/spanner-common/src/test/java/com/google/cloud/teleport/v2/spanner/migrations/avro/AvroToValueMapperTest.java

```diff
@@ -168,6 +168,68 @@ public void testAvroFieldToDouble_UnsupportedType() {
     AvroToValueMapper.avroFieldToDouble(inputValue, SchemaBuilder.builder().booleanType());
   }
 
+  @Test
+  public void testAvroFieldToFloat_ValidFloat32Input() {
+    Float inputValue = 5.75f;
+    Float result =
+        AvroToValueMapper.avroFieldToFloat32(inputValue, SchemaBuilder.builder().doubleType());
+    assertEquals(inputValue, result);
+
+    /* Test a valid double to float mapping */
+    result =
+        AvroToValueMapper.avroFieldToFloat32((Double) 3.14, SchemaBuilder.builder().floatType());
```

Comment:
should this be Float instead of Double ?

Reply:
This is `double` on purpose. As we have discussed in comments, we test cross type mapping of a double input as a float output - this is to handle the case the source and destination schemas happen  to be double->float32